### PR TITLE
Added convenience functionality to 2D analysis code.  -MW

### PR DESCRIPTION
**Pull request by matthewware**

### CODE REVIEW

**Comment by dieris** [comment]
can I merge it with https://github.com/BBN-Q/Qlab/pull/62?

**Comment by matthewware** [comment]
Sure!  Looks good to me.

**Comment by dieris** [comment]
@matthewware can you change your PR target to the branch in  #62 ? So that I merge them with each other before develop

**Comment by matthewware** [comment]
@dieris is this what you were thinking?  I had to merge the two PR's since they were both upstream.

File: analysis/cQED_dataAnalysis/fitrabi.m [hunk 1]
```diff
@@ -30,8 +29,8 @@
 % model A + B Exp(-t/tau) * cos(w t + phi)
 rabif = inline('p(1) + p(2)*exp(-tdata/p(3)).*cos(p(4)*tdata + p(5))','p','tdata');
 
-% initial guess for amplitude is y(1) - mean
-amp = y(1) - mean(y);
+% initial guess for amplitude is y(max) - mean
+amp = y(max) - mean(y);
```
Comment (caryan) [inline analysis/cQED_dataAnalysis/fitrabi.m]: @dieris this seems like a typo maybe `max(y)`?

Comment (dieris) [inline analysis/cQED_dataAnalysis/fitrabi.m]: yes, sorry I meant max(y)

Comment (matthewware) [inline analysis/cQED_dataAnalysis/fitrabi.m]: Sorry about that.  It was one of my repos.  I've pushed it to Qlab as feature/2Dimprovements.  Should I rebase on develop?

Comment (dieris) [inline analysis/cQED_dataAnalysis/fitrabi.m]: yes, thanks, that should work

**Comment by caryan** [comment]
That was nice how GitHub figured out to close this PR.

**Pull request merged**